### ISSUE TITLE
issue with pull

### ISSUE THREAD

**Issue by arnos** [hwuethrich/docker-images#13]
Using coreOS 494 (panamax) Docker 1.3.2 

When I pull your image 
```
docker pull hwuethrich/bamboo-server:latest
```

I get the following error
```
Pulling repository hwuethrich/bamboo-server
25e33a769d1a: Error pulling image (latest) from hwuethrich/bamboo-server, ApplyLayer exit status 1 exec: "xz": executable file not found in $PATH
8dbd9e392a96: Error downloading dependent layers
2015/01/26 15:15:50 Error pulling image (latest) from hwuethrich/bamboo-server, ApplyLayer exit status 1 exec: "xz": executable file not found in $PATH
```

xz is available on the host and other images download without an issue (and I've tested on older version of CoreOS it works without an issue)

Maybe it's just a question of rebuilding the image (10 month old)

**Comment by nichaladas** [comment]
This looks like a known issue with Docker 1.3.2.  I just ran into the same problem.

https://github.com/docker/docker/issues/9327

**Comment by hwuethrich** [comment]
Yes, it seems to be solved with 1.3.3. Thanks for reporting anyway. I'm not using this image anymore (because I switched job) but would like to upgrade it to use something like [/phusion/baseimage-docker](phusion/baseimage-docker) soon.

**Comment by hwuethrich** [comment]
FYI, I pushed a new version of the image which is now based on [phusion/baseimage-docker](/phusion/baseimage-docker). It also installs Bamboo Server 5.7.2 by default now. Let me know if this causes any issues.
